src/app.ts

```diff
@@ -74,6 +74,12 @@ export class App {
       this.handleOnOff(true);
     });
 
+    this.eventAggregator.subscribe("firebase.signature", async (onOff: boolean) => {
+      const signatureMessage = "Check your wallet provider to ensure it is not waiting for your signature.";
+      this.modalMessage = this.modalMessage ? `${this.modalMessage.substring(0, this.modalMessage.length - 2)} ${signatureMessage}` : signatureMessage;
```

Comment:
I don't think you should base your message on `this.modalMessage` because you don't necessarily know what it is.  I would hard code the message.